tests/Integration/Bridge/TolerantParser/ReferenceFinder/NameSearcherCompletorTest.php

```diff
@@ -27,51 +25,69 @@ public function testComplete(string $source, array $expected): void
     public function provideComplete(): Generator
     {
         yield 'class' => [
-            '<?php class Foobar {} :int {}; new Foo<>', [
+            '<?php class Foobar { public function __construct(int $cparam) {} } :int {}; new Foo<>', [
                 [
-                    'type' => Suggestion::TYPE_CLASS,
-                    'name' => 'Foobar',
+                    'type'              => Suggestion::TYPE_CLASS,
+                    'name'              => 'Foobar',
                     'short_description' => 'Foobar',
+                    'snippet'           => 'Foobar(${1:\\$cparam})${0}',
```

Comment:
as the CS fixer doesn't pick this up I won't block it, but in general there is no whitespace alignment in Phpactor.

Reply:
Now I know. Maybe this should be added then to the CS Fixer.